x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -9,6 +9,7 @@ import { i18n } from '@kbn/i18n';
 
 import { get } from 'lodash';
 import { LicenseType } from '../../../../licensing/common/types';
+import { getCasesDeepLinks } from '../../../../cases/public';
```

Comment:
I think is best to provide the utility function through the `CasesUIClient`. Security solution should pass the `getCasesDeepLinks` as a dependency to `getDeepLinks`. Something like `casesUIClient.getCasesDeepLinks` or `casesUIClient.utils.getCasesDeepLinks`.

Because `getCasesDeepLinks` will be needed in the setup lifecycle of Security solution we should exported also from there.

Reply:
As we discussed we will do it in another PR